types/objc/type_encoding.go

```diff
@@ -229,7 +229,7 @@ func getPropertyAttributeTypes(attrs string) (string, bool) {
 		case propertyWeak:
 			attrsList = append(attrsList, "weak")
 		case propertyDynamic:
-			attrsList = append(attrsList, "@dynamic")
+			// attrsList = append(attrsList, "@dynamic")
```

Comment:
Go doesn't fallthrough by default so instead of just commenting this out, we should instead have a comment saying why it doesn't need to be represented in the objc class-dump